src/Fabulous.XamarinForms/Views/Controls/ProgressBar.fs

```diff
@@ -33,6 +49,16 @@ type ProgressBarModifiers =
     static member inline progressColor(this: WidgetBuilder<'msg, #IProgressBar>, light: Color, ?dark: Color) =
         this.AddScalar(ProgressBar.ProgressColor.WithValue(AppTheme.create light dark))
 
+    [<Extension>]
+    static member inline progressTo
```

Comment:
I think it would be better to use it through the constructor. Because what happens if I do?
```fs
ProgressBar(100)
    .progressTo(25, 100, Easing.Linear)
```

Instead if it was an additional constructor, we could do
```fs
// Not animated
ProgressBar(model.Value)

// Animated
ProgressBar(model.Value, 100, Easing.Linear)

```

Reply:
yeah that make sense . Will update it ;)

Reply:
Updated following your suggestion and also updated the video . Thanks for the review ;)